From: Lukas Schwaighofer <lukas@schwaighofer.name>
Date: Sun, 11 Nov 2018 10:09:35 +0100
Subject: Disable tests requiring Internet access.

Tests requiring Internet access will fail on Debian's autobuilding
machines.

Forwarded: not-needed
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e0f338..dd28b74 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,13 +84,13 @@ ADD_TEST(test_trie tests/test_trie)
 ADD_TEST(test_ht_spkitable tests/test_ht_spkitable)
 ADD_TEST(test_ht_spkitable_locks tests/test_ht_spkitable_locks)
 
-ADD_TEST(test_live_validation tests/test_live_validation)
+#ADD_TEST(test_live_validation tests/test_live_validation)
 
 ADD_TEST(test_ipaddr tests/test_ipaddr)
 
 ADD_TEST(test_getbits tests/test_getbits)
 
-ADD_TEST(test_dynamic_groups tests/test_dynamic_groups)
+#ADD_TEST(test_dynamic_groups tests/test_dynamic_groups)
 
 #install lib
 set (RTRLIB_VERSION_MAJOR 0)
